### ISSUE TITLE
[Expression] Support DateTimeDiff in prebuilt functions

### ISSUE THREAD

**Issue by cosmicshuai** [microsoft/botbuilder-dotnet#3933]
DateTimeDiff function works as
```
DateTimeDiff(Date1: string, Date2: string) => ticks: number
```

**Comment by boydc2014** [comment]
on hold, this depends on #3945

**Comment by cosmicshuai** [comment]
#3945 has been merged.
#3934 has been updated.
